### PR TITLE
Desktop Version Portfolio

### PR DESCRIPTION
**Pull request by mgmediaweb**
# In this pull request includes the following actions:

- Optimized CSS for the desktop version.
- Organized CSS for Linter Rules.
- Visually in the browser everything looks perfect and without design errors.
- Testing with local Linters and all passed.

### CODE REVIEW

File: Assets/css/style.css [hunk 1]
```diff
@@ -463,91 +487,23 @@ article:hover .work-btn {
...
   cursor: pointer;
 }
 
-/* Setup for mobile devices with screen is 500px */
+/* XXXXXXXXXXXXXXXXXXXXXXXXXXX  mobile devices  XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX  */
 
 @media only screen and (max-width: 500px) {
   nav {
     display: none;
   }
```
Comment (AmaduKamara) [inline Assets/css/style.css:501]: I was a big fan of styling for desktop to mobile screens too, but for good convention and good practices, it's almost a must in the modern web development era to start your styles from mobile to desktop screens.

- [ ] Kindly style your website from Mobile to desktop screens as it's required for this project your styles start from mobile to desktop screens and are in the range from 768px and above but not above 1440px.  I suggest you start all your desktop screen styles from 768px above.
- Hint: 
You can put all your desktop screen styles in this media query:
```mdx
@media screen and (min-width: 768px) { ... }
```
and all other styles can be left for mobile screens. 👍

File: Assets/css/style.css [hunk 2]
```diff
@@ -65,6 +63,114 @@ textarea {
...
+  position: relative;
+  width: 100%;
+}
+
+.main-presentation {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
```
Comment (AmaduKamara) [inline Assets/css/style.css:141]: - [ ] Kindly adjust your styles to slightly move the left text to align with the Logon on the navbar in vertical view.

- From the look of your design below things are not aligned properly on desktop screens

image

- See how things are aligned in the Figma design.
image

- [ ] Please fix that 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @mgmediaweb,

AlmostThereGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Github flow is set up correctly
✅ No linter errors
✅ Good descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I kindly recommend you take all the styles for the desktop screens and place them inside this media query below. 👍 You can do it better.
```mdx
@media screen and (min-width: 768px) { ... }
```

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by mgmediaweb** [comment]
Update all issues, please check the changes...

**Comment by AmaduKamara** [comment]
Good job working on the changes. Your project looks great now and how do you feel about making the changes? 👍 
Please send it for a review

**Review by gtekle — APPROVED** [review]
Hi @mgmediaweb :wave: , 

## STATUS: APPROVED 🎉 🎉 🎉:clap: :clap: :clap: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

mazel tov

## Highlights
:heavy_check_mark: Well organized and neat code
:heavy_check_mark: Nice UI design
:heavy_check_mark: Good use of Githu flow

## Optional suggestions
[Optional] You should check the headline with screen sizes between 769px to 1080px. I believe the background image is too big for this range.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**